jina/drivers/search.py

```diff
@@ -20,12 +20,12 @@ class BaseSearchDriver(BaseExecutableDriver):
     """Drivers inherited from this Driver will bind :meth:`query` by default """
 
     def __init__(
-        self,
-        executor: Optional[str] = None,
-        method: str = 'query',
-        traversal_paths: Tuple[str] = ('r', 'c'),
-        *args,
-        **kwargs,
+            self,
```

Comment:
The formatting change is odd. Is that the result of black? It shouldn't happen anymore

Reply:
Did not pass black check yesterday.